Package.swift

```diff
@@ -51,19 +51,25 @@ let packageDependencies: [Package.Dependency] = [
     from: "1.20.2"
   ),
   .package(
-    url: "https://github.com/apple/swift-log.git",
-    from: "1.4.4"
+    // url: "https://github.com/apple/swift-log.git",
+    // from: "1.4.4"
+    url: "https://github.com/slashmo/swift-log.git",
```

Comment:
We'd also need these changes in `Package@swift-5.5.swift`

Reply:
Right, gotcha